server/routes/topic_page.py

```diff
@@ -16,25 +16,42 @@
 from flask import current_app
 import flask
 import os
+import json
 import routes.api.place as place_api
 from google.protobuf.json_format import MessageToJson
 
 bp = flask.Blueprint('topic_page', __name__, url_prefix='/topic')
 
 
 @bp.route('', strict_slashes=False)
+@bp.route('/<string:topic_id>', strict_slashes=False)
 @bp.route('/<string:topic_id>/<path:place_dcid>', strict_slashes=False)
 def topic_page(topic_id=None, place_dcid=None):
     if os.environ.get('FLASK_ENV') == 'production':
         flask.abort(404)
 
+    topics_summary = json.dumps(get_topics_summary())
     # Redirect to the landing page.
-    if not place_dcid or not topic_id:
+    if not place_dcid and not topic_id:
         return flask.render_template('topic_page_landing.html')
 
-    # Find the config for the topic & place.
     all_configs = current_app.config['TOPIC_PAGE_CONFIG']
```

Comment:
we could use topics_summary for this too

Reply:
I don't think we can use the summary here because the summary doesn't have the actual configs

Reply:
ah yes, good point.